posts/inside-rust/2020-01-27-Cleanup-Crew-ICE-breakers.md

```diff
@@ -32,7 +32,7 @@ This kind of cleanup is invaluable in getting bugs fixed.
 
 It can be done by anybody who knows Rust, without any particularly deep
 knowledge of the compiler.  If you want to be part of it and be notified
-about things to do, just [open a PR]! When we come across a suitable
+about things to do, just add yourself to the list ([instructions here])! When we come across a suitable
```

Comment:
Generally "click here" or similar is bad content on the web as it doesn't describe what is behind the text. Putting the link around the action you want users to perform is more descriptive. 
```suggestion
about things to do, just [add yourself to the list][instructions here]! When we come across a suitable
```